pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -68,19 +67,15 @@
         BetaToolUseBlock,
         BetaToolUseBlockParam,
     )
+    from anthropic.types.model_param import ModelParam
 
 except ImportError as _import_error:
     raise ImportError(
         'Please install `anthropic` to use the Anthropic model, '
         'you can use the `anthropic` optional group — `pip install "pydantic-ai-slim[anthropic]"`'
     ) from _import_error
 
-LatestAnthropicModelNames = Literal[
-    'claude-3-7-sonnet-latest',
-    'claude-3-5-haiku-latest',
-    'claude-3-5-sonnet-latest',
-    'claude-3-opus-latest',
-]
+LatestAnthropicModelNames = ModelParam
```

Comment:
@samuelcolvin did you on purposely not use this alias to make it a better user experience?